content/guides/getting-started.md

```diff
@@ -218,18 +235,40 @@ Given it's not particularly fun to run a local copy of webpack from the CLI, we
 
 Now the `npm run build` command can be used in place of the longer commands we used earlier. Note that within `scripts` we can reference locally installed npm packages by name instead of writing out the entire path. This convention is the standard in most npm-based projects and allows us to directly call `webpack`, instead of `node_modules/webpack/bin/webpack.js`
 
+Now run the following command and see if your script alias works:
```

Comment:
You'll need to add a break between this line and the code block to please our markdown linter.